src/sass/components/_nested-navigation.scss

```diff
@@ -0,0 +1,127 @@
+.#{$prefix}-nested-nav {
+  &__list {
+    flex-basis: 100%;
+    list-style-type: none;
+    padding-left: 0;
+  }
+
+  &__list[hidden],
+  &__list[aria-hidden='true'] {
+    display: none;
+  }
+
+  &__item {
+    align-items: center;
+    border-top: 1px solid $color-black--100;
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 0; // resets Rivet default margin-top
+  }
+
+  &__link {
+    color: $color-black;
+    flex-grow: 1;
+    padding: $sm;
+    text-decoration: none;
+
+    &[aria-current] {
+      box-shadow: inset $xxs 0 0 $color-blue;
+    }
+
+    &:focus,
+    &:hover {
+      color: $color-blue;
+      text-decoration: underline;
+    }
+  }
+
+  &__icon {
+    align-items: center;
+    background: transparent;
+    border: none;
+    justify-content: center;
+    padding: $sm;
+
+    svg {
+      vertical-align: middle;
+    }
+
+    &[aria-expanded="true"] {
+      transform: rotate(180deg);
+    }
+
+    &:focus {
+      outline: $xxs/2 solid $color-blue;
+      outline-offset: $xxs/2;
+    }
+
+    &:focus,
+    &:hover {
+      background: $color-blue;
+      color: $color-white;
+    }
+  }
+
+  &__link,
+  &__icon {
+    &:focus {
+      z-index: 1;
+    }
+  }
+}
+
+/*
+  * Nested levels:
+  * Indent text and set background colors and border-top colors
+  */
+$list: 'rvt-nested-nav__list';
+$item: 'rvt-nested-nav__item';
+
+/* Set up initial selectors for indenting and shading */
+$indent-selector: '.#{$list} > .#{$item} > .#{$list} > .#{$item}';
+$shade-selector: '.#{$item} > .#{$list}';
+
+// Set up level indent/background shade map
+$level-styles: (
+  40: $color-black--050,
```

Comment:
Let's cut these indentation values down by half. When you think of how this component will generally be used as a sidenav/sidebar, in those deeper nested levels you might barely have enough width for one word, and if it's a long word you're going to run into one of those _CSS IS AWESOME_ situations where the text overflows outside of the link/item or else expands the width of the whole (`.rvt-nested-nav`) parent container.